README.md

```
@@ -8,6 +8,18 @@ This is Cole's version with a couple of improvements and the addition of Supabas
 Also, the local RAG AI Agent workflows from the video will be automatically in your 
 n8n instance if you use this setup instead of the base one provided by n8n!
 
+> [!IMPORTANT]
+> **Pulling the latest changes after having the package running already?** Supabase keeps moving environment variables out of its Docker Compose file and into `.env`, so you may need to add new defaults that are already in my `.env.example`. Most recently, the Storage service started requiring its own config in `.env` — if it's missing, the `supabase-storage` container crashes on startup with a `region is missing` error. Add the following to your `.env`:
+>
+> ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)